invokeai/backend/install/invokeai_configure.py

```diff
@@ -654,10 +657,13 @@ def migrate_init_file(legacy_format: Path):
     old = legacy_parser.parse_args([f"@{str(legacy_format)}"])
     new = InvokeAIAppConfig.get_config()
 
-    fields = list(get_type_hints(InvokeAIAppConfig).keys())
+    fields = [x for x, y in InvokeAIAppConfig.__fields__.items() if y.field_info.extra.get("category") != "DEPRECATED"]
```

Comment:
This seems pretty fiddly, when all that really needs to happen is we ignore the old unused fields from the config when it is parsed into the pydantic model.

By default, pydantic just ignores values that are not defined fields. You can change this behaviour with `extra` as described here: https://docs.pydantic.dev/1.10/usage/model_config/#options

Because the default is to ignore extra values, you can just delete the deprecated fields from the model definition.

I'm not sure if there is any other intention here with marking things deprecated, and then ignoring them.